src/handlers.js

```diff
@@ -41,9 +41,8 @@ const createReportHandlers = (reportTemplates) => {
         const params = [...requiredParams, ...optionalParams];
 
         // Build the optional param default object here (once) for later use
-        const optionalParamDefaults = Object.assign(
-            {}, ...optionalParams.map((p) => ({ [p]: template.dataSource.params[p].default })),
-        );
+        // eslint-disable-next-line max-len
+        const optionalParamDefaults = Object.assign({}, ...optionalParams.map((p) => ({ [p]: template.dataSource.params[p].default })));
```

Comment:
This might be a little nicer as
```suggestion
        const optionalParamDefaults = Object.fromEntries(optionalParams.map((p) => ([p, template.dataSource.params[p].default });
```
(I probably didn't close all the correct brackets etc. here)